junit/src/main/java/org/kaazing/k3po/junit/rules/Latch.java

```diff
@@ -93,7 +93,7 @@ void notifyFinished() {
         switch (state) {
         case INIT:
             notifyPrepared();
-            break;
+//            break;
```

Comment:
notifyPrepared might be needed in case the main thread is in awaitPrepared and there is an abort coming right after preparing the script. Something like this might occur in case we have an assumeTrue or assumeFalse. If we remove it, the main thread will remain blocked.

The FINISHED event can come after an ABORT. And the ABORT can be send even if a START was not sent. We have also the situation when the script contains only servers, but it does not make sense from testing point of view.
The spec is already specifying that the FINISHED can come after an ABORT and an ABORT before the START.
`Any time after the PREPARE command and before DISPOSE, the test framework can send an **ABORT** command. If the execution of the scripts is already finished, no event is sent back. If the execution is aborted by this command, a FINISHED event with the script that was executed so far must be returned.`
So we should still keep it this way.